packages/salesforcedx-sobjects-faux-generator/test/fauxClassGenerator.test.ts

```diff
@@ -24,6 +24,43 @@ describe('SObject faux class generator', function() {
     }
   });
 
+  it('Should generate a faux class with a proper header comment', async () => {
+    const fieldsHeader = '{ "name": "Custom__c", "fields": [ ';
+    const closeHeader = ' ], "childRelationships": [] }';
+
+    const sobject1 = `${fieldsHeader}${closeHeader}`;
```

Comment:
Why bother splitting this into 2 variables?

Reply:
because the other tests were already set up that way - they have extra fields that weren't necessary for this test.  Seemed simpler to keep the same pattern rather that do something special here.

Reply:
Having 3 separate variables for 1 JSON string is more convoluted than being consistent across various unit tests.